applications/asset_tracker_v2/overlay-carrier.conf

```diff
@@ -6,6 +6,10 @@
 
 CONFIG_LWM2M_CARRIER=y
 
+# Workaround for a partition manager bug where multiple partitions are defined in the non-secure
+# storage. By increasing the size of the LwM2M carrier storage to fill 32 kB we circumvent the bug.
+CONFIG_LWM2M_CARRIER_STORAGE_AREA_SIZE=0x8000
+
```

Comment:
I would have put Settings storage to the same size as well, and maybe document that you are putting the storage to the size of the SPU region size. Since this workaround would have to be 16 KB for nrf5340 for example.

For reference this is the known issue text that has been added:
https://github.com/nrfconnect/sdk-nrf/pull/8469/files#diff-a735de3a5c2dadf8c6e9cb659c0a3b14deb436c036827a2880665b9059d6f5f4R2359

Reply:
Thanks. It looks like it's not necessary to increase the settings storage, so I'd rather not. Have updated the comment to mention the SPU region size.